### ISSUE TITLE
Alpha Strike/Battle Force

### ISSUE THREAD

**Issue by Maelwys** [Solaris-Skunk-Werks/solarisskunkwerks#60]
Alpha Strike seems to have taken over Battle Force completely. From what I understand, they're basically the same thing, but Alpha Strike is at the Mech level, while Battle Force seems more dedicated to Lance and above, so i suggest renaming everything BattleForce as Alpha Strike, and update the rules to take into account the errata that's come out.

https://bg.battletech.com/wp-content/uploads/2019/09/Alpha-Strike-Companion-Converting-Heat-Errata-v1.2.pdf

The big thing seems to be the way heat is handled, especially with regards to XXL engines, as well as certain pieces of equipment listed in the errata (Laser AMS).

The MAL-YZ Malice seems to be a good test subject in this case. 

Right now SSW seems to be missing the ARM, CR, OVL, RSD2, RCN and TAG codes (SRCH, SOA, SEAL and ES are things all mechs have..the MUL omits it since all BMs have it, SSW includes it for completeness sake).

**Comment by WEKarnesky** [comment]
This is at least partially in progress, as there is an "Alpha Strike" printing option (at least in BFB).  For this reason I tagged it as both a bug and a feature ticket.

I think Battleforce is still in the current rules printings, If so I do not want to remove it, but we should definitely Add Alphastrike everywhere it is missing.

**Comment by Maelwys** [comment]
This is what the Forums have to say about it. Basically Alpha Strike and Battle Force have the same basic unit stats, its just how they're eventually used. 

Since Alpha Strike stays with the individual Mech, while Battle Force is down to the lance level, maybe we should change the tab to Alpha Strike?

https://bg.battletech.com/forums/index.php?topic=67131.msg1551975#msg1551975

**Comment by WEKarnesky** [comment]
That was the first I had heard that, so I spoke to Alex Knight, and confirmed that to be the case.

  The only difference between AS and BF is that AS uses inches at two times the hex movement.  If we support both inch and hex moment printing then the output will be fully compatible with both.

Do we have the space to change the wording to "Alpha Strike/Battle Force" from everywhere that is just "Battle Force"?

**Comment by Maelwys** [comment]
The point values are also calculated differently I believe.

Maybe we could change things to A.S/B.F. ?

**Comment by WEKarnesky** [comment]
Alex Knight says that the Point Values are the same for both.

What happened is that the very first version of AS used BF conversion rules from SO, then they updated them and SO will sometime have errata to match. Or, in a more perfect world BF in SO would just say "For Battleforce units, take the Alpha Strike values for each lance member"

**Comment by Maelwys** [comment]
Ah. Well. That's interesting. So yeah. I guess an option to show if you want the movement done in hexes or inches, and incorporating errata (and special abilities) is "all' that's needed.

Atleast it isn't from the ground up.

**Comment by Notsonoble** [comment]
Since I'm rebuilding print templates, should they say:
Alpha Strike
Alpha Strike/BattleForce
BattleForce for printing folder, and Alpha Strike in the AS_Card folder?

**Comment by WEKarnesky** [comment]
Great question.  I would say universalize it to `Alpha Strike/BattleForce` where there is room, and `Alpha Strike` where there isn't.

As the rules are identical to make the cards, I don't think we need to separate them out If someone has other opinions or think I'm missing something, I'm open and eager to hear them.

**Comment by RandallBurt** [comment]
Late to this party, but Alpha Strike TMM is missing from the cards and afaik, they did change the point value calculation from BF2 to Alpha Strike. The point values don't match the MUL.

**Comment by WEKarnesky** [comment]
That is correct @RandallBurt. I am under the impression that damage calculations are also off and the special abilities aren't correct either.  I want to go over it with a fine tooth comb with the eratta, but I have been busy with some real life events.  I saw you forked, if you develop feel free to join us on the discord, but either way if you fix any bugs, we are happily accepting pull requests.

**Comment by RandallBurt** [comment]
I haven't seen any damage inconsistencies in any of the units I've checked so far, *but* there are so many fiddly bits with the conversions I wouldn't be surprised. I'm going to pick up the companion this weekend and take a look at the code some. My Java is rusty and I also have IRL stuff going on but this tool is super handy so hoping I can jump in and help.

**Comment by WEKarnesky** [comment]
We're more than happy to have your help.  Feel free to ask any questions in our Discord https://discordapp.com/invite/xc5pUWP

**Comment by HyaeksVers** [comment]
Any updates on this issue? AS PV values are still inconsistent from MUL
